extensions/azurePublishNew/src/node/availableResources.ts

```diff
@@ -19,6 +19,13 @@ import { getLuisPredictionProvisionService, luisPredictionDefinition } from './a
 import { getQnAProvisionService, qnaDefinition } from './azureResources/qna';
 import { getAppServiceProvisionService, servicePlanDefinition } from './azureResources/servicePlan';
 import { getWebAppProvisionService, webAppResourceDefinition } from './azureResources/webApp';
+import { ProvisionConfig2 } from './provisioning';
```

Comment:
ProvisionConfig2 is the required input to the provisioning system from the UI.  The current ProvisionConfig should probably be renamed (ProvisionServiceConfig?) and ProvisionConfig2 should be ProvisionConfig.